File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) 
    {
        unordered_map<int,int> umap;
        priority_queue<pair<int,int>> answer;
        for(int i = 0; i < nums.size();i++)
        {
            umap[nums[i]]++;
        }
        for(auto p : umap)
        {
            answer.push({p.second,p.first});
        }
        vector<int> answer2;
        for(int i = 0; i < k;i++)
        {
            answer2.push_back(answer.top().second);
            answer.pop();
        }
        
        return answer2;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        umap = {}
        for n in nums:
            umap[n] = 1
        maxL = 0
        for n in nums:
            i = 1
            length = 1
            while n+i in umap:
                length += 1
                i += 1
            if maxL < length:
                maxL = length
        return maxL


        

        